abce/multiprocess.py

```diff
@@ -47,7 +47,7 @@ def insert_or_append(self, group, ids, Agent, simulation_parameters, agent_param
                 agent = Agent(id, simulation_parameters, ap, **agent_arguments)
                 agent._send = agent._send_multiprocessing
                 agent._out = defaultdict(list)
-                agent.init(simulation_parameters, ap)
+                agent.init(**ChainMap(simulation_parameters, ap))
```

Comment:
Why is the line not `agent.init(ap, **simulation_parameters)`?

Reply:
From this line, my understanding is that even `agent_parameters` is no longer a required argument?